cmake/config.cmake

```diff
@@ -82,6 +85,10 @@ set(OMR_TOOLS_IMPORTFILE "IMPORTFILE-NOTFOUND" CACHE FILEPATH
 	"Point it to the ImportTools.cmake file of a native build"
 )
 
+set(OMR_TOOLS_USE_NATIVE_ENCODING FALSE CACHE BOOL
```

Comment:
This should be more consistent: use `OFF` / `ON` or `FALSE` / `TRUE` instead of a mixture.

Reply:
Fixed in b7ecc9a to use `OFF` / `ON` as the rest of the file uses the same.